sklearn/compose/tests/test_column_transformer.py

```diff
@@ -120,15 +135,75 @@ def test_column_transformer():
                      transformer_weights['trans2'] * X_res_second1D]).T
     assert_array_equal(both.fit_transform(X_array), res)
     assert_array_equal(both.fit(X_array).transform(X_array), res)
+    assert_allclose(both.inverse_transform(res), X_array)
     assert len(both.transformers_) == 2
 
     both = ColumnTransformer([('trans', Trans(), [0, 1])],
                              transformer_weights={'trans': .1})
     assert_array_equal(both.fit_transform(X_array), 0.1 * X_res_both)
     assert_array_equal(both.fit(X_array).transform(X_array), 0.1 * X_res_both)
+    assert_allclose(both.inverse_transform(0.1 * X_res_both), X_array)
     assert len(both.transformers_) == 1
 
 
+def test_column_transformer_inverse_transform_all_transformers_drop():
+    X_array = np.array([[0, 1, 2], [2, 4, 6]]).T
+
+    # TransNo2D does not define inverse_transform
+    ct = ColumnTransformer([('trans1', 'drop', [0]),
+                            ('trans2', 'drop', [1])])
+    res = ct.fit_transform(X_array)
+    assert_array_equal(np.zeros((X_array.shape[0], 0)), res)
+
+
+def test_column_transformer_inverse_transform_with_drop():
```

Comment:
These error tests could be one test with `pytest.mark.parametrize` defining both the constituent estimators and the error message.